README.md

```diff
@@ -2,7 +2,17 @@
 
 # CloudWeGo
 
-The [cloudwego.github.io](https://cloudwego.github.io/) site, built using [Hugo](https://gohugo.io/) and hosted on [Github Pages](https://pages.github.com/).
+The [cloudwego.io](https://www.cloudwego.io/) site, built using [Hugo](https://gohugo.io/) and hosted on [Github Pages](https://pages.github.com/).
```

Comment:
"The xxx site is built with hugo and hosted on Github Pages." Looks Better in this way ？